src/main/scala/net/ripe/rpki/publicationserver/store/postgresql/PgStore.scala

```diff
@@ -31,7 +31,7 @@ class PgStore(val pgConfig: PgConfig) extends Hashing with Logging {
   def getState = DB.localTx { implicit session =>
     sql"SELECT * FROM current_state ORDER BY url"
       .map { rs =>
-        val hash = Hash(rs.string(1))
+        val hash = Hash(rs.bytes(1))
```

Comment:
Can this be by name?

Reply:
Probably not since we need to get the bytes while the database cursor is on the current row. Also, the data is only 32 bytes here. We should probably remove the `SELECT *` though and use explicit column names or get the bytes by column name. Using a position index to fetch the column is a bit error prone here.